spec/requests/api/v0/webhooks_spec.rb

```diff
@@ -57,18 +66,16 @@
       end.to change(Webhook::Endpoint, :count).by(-1)
     end
 
-    it "returns 200 on success" do
+    it "returns 204 on success" do
       delete "/api/webhooks/#{webhook.id}"
-      expect(response).to have_http_status(:ok)
-      expect(response.content_type).to eq("application/json")
-      json = JSON.parse(response.body)
-      expect(json["success"]).to be true
+      expect(response).to have_http_status(:no_content)
     end
 
     it "doesn't allow to destroy other user webhook" do
       other_webhook = create(:webhook_endpoint, user: create(:user))
       expect do
         delete "/api/webhooks/#{other_webhook.id}"
+        expect(response).to have_http_status(:not_found)
```

Comment:
I'm a bit uncomfortable about having 2 expectations that check unrelated things. How do you feel about extracting response expectation to a separate test?

Reply:
Sure, I gave up a little on that because most of our specs contain multiple expectations, I'll extract it!